src/Telemetry/AzureDevOpsTimeline/ITimelineTelemetryRepository.cs

```diff
@@ -11,8 +11,8 @@ namespace Microsoft.DotNet.AzureDevOpsTimeline
 {
     public interface ITimelineTelemetryRepository : IDisposable
     {
-        public Task<DateTimeOffset?> GetLatestTimelineBuild(string project);
-        public Task WriteTimelineBuilds(IEnumerable<AugmentedBuild> augmentedBuilds);
+        public Task<DateTimeOffset?> GetLatestTimelineBuild(AzureDevOpsInstance instance);
```

Comment:
Same thing here, this feels like I shouldn't have to pass the client in here.  Especially since "WriteTimelineBuilds" takes an organization, can't this _also_ just take the org and let me use that?

Reply:
Ooo. It's the naming. A "project" isn't really an "Instance".  It's... well... a project. Can we just call this something like "AzureDevOpsProjectId" or something?

Reply:
its not a project though. Its "org/project'

Reply:
Well, I mean, it _is_ a project. The name of the project by itself is useless in 100% of contexts, you _always_ need the org to fully qualify it.  It's more a "project" than an "instance" at least. :-) Since it's not actually an instance of anything (it's just some identifiers).  You call it FullyQualifiedAzureDevOpsProjectId... but that's a bit verbose, and "ProjectId" by itself is meaningless.

Reply:
I changed it.